fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -37,6 +37,9 @@
     private static final Logger LOGGER = LoggerFactory
             .getLogger(FedoraUserSecurityContext.class);
 
+    // Defined here because FedoraJcrTypes is in fcrepo-http-api, which would create a cyclic dependency
+    protected static final String JCR_CONTENT = "jcr:content";
```

Comment:
We're talking about different things.

This PR creates a dependency on `fcrepo-kernel-api`, not `fcrepo-http-api`. If the dependency to `fcrepo-kernel-api` is added to maven (which it should be: see my additional commit), there is no complaint. This was the whole point, a few weeks ago, in removing `fcrepo-mint` from the kernel-api: that way the kernel-api is a the very bottom of food-chain, as it were, and *any* other module should freely be able to use it.